python/src/multi_agent_orchestrator/agents/parallel_agent.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+from typing import Any, AsyncIterable
+
+from multi_agent_orchestrator.agents import (
+    Agent,
+    AgentOptions,
+    BedrockLLMAgent,
+)
+from multi_agent_orchestrator.types import ConversationMessage, ParticipantRole
+from multi_agent_orchestrator.utils.logger import Logger
+
+
+# Extend AgentOptions for ParallelAgent class:
+class ParallelAgentOptions(AgentOptions):
+    def __init__(
+        self,
+        agents: list[str],
+        default_output: str = None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.agents = agents
+        self.default_output = default_output
+
+
+# Create a new custom agent that allows for parallel processing:
+class ParallelAgent(Agent):
+    def __init__(self, options: ParallelAgentOptions):
+        super().__init__(options)
+        self.agents = options.agents
+        self.default_output = (
+            options.default_output or "No output generated from the ParallelAgent."
+        )
+        if len(self.agents) == 0:
+            raise ValueError("ParallelAgent requires at least 1 agent to initiate!")
+
+    async def _get_llm_response(
+        self,
+        agent: BedrockLLMAgent,
```

Comment:
Change from BedrockLLMAgent to Agent

Reply:
Fixed in latest commit below: [Update parallel_agent.py after initial PR review](https://github.com/awslabs/multi-agent-orchestrator/pull/103/commits/42f064e90b1c796cf39c9db8174bc932124b0c3b)